frontend/src/components/editor/chrome/components/feedback-button.tsx

```diff
@@ -66,71 +55,53 @@ const FeedbackModal: React.FC<{
         <DialogHeader>
           <DialogTitle>Send Feedback</DialogTitle>
           <DialogDescription>
-            Let us know what you think about marimo! If you have a bug that you
-            would like to report, please use the{" "}
-            <a
-              href={Constants.issuesPage}
-              target="_blank"
-              rel="noreferrer"
-              className="underline"
-            >
-              GitHub issue tracker
-            </a>
-            .
-          </DialogDescription>
-          <DialogDescription>
-            Your feedback is anonymous and will help us improve marimo. Thank
-            you!
+            <p className="my-2 prose">
```

Comment:
You might be able to hoist the prose class higher - you need dark mode too: `dark:prose-invert`

Reply:
added dark, hoisting didn't seem to work